src/lib.rs

```diff
@@ -7,6 +7,8 @@ use std::str::Utf8Error;
 extern crate bitflags;
 #[macro_use]
 extern crate enum_primitive_derive;
+#[macro_use(AsRefStr)]
+extern crate strum_macros;
```

Comment:
I think `extern crate` with `macro_use` is deprecated, and the `strum_macros` docs seem to agree:
`use strum_macros::{AsRefStr};`

It would be nice to fix the other `extern crate`s at the same time.

Reply:
Done. Pushed as a new commit.